styles.css

```diff
@@ -0,0 +1,84 @@
+/* toollbar styles */
+
+.header {
```

Comment:
it's good, but Webix already has a predefined styling for a dark toolbar - css: "webix_dark"